backend/testing/common/showmgt/handler.create-movie_test.go

```
@@ -1,196 +1,195 @@
+// //         }`)))
+
+// // 		router.ServeHTTP(recorder, testutils.WithFakeJWT(request))
+
+// // 		var response core.Response[any]
+// // 		_ = json.Unmarshal(recorder.Body.Bytes(), &response)
+
+// // 		Expect(recorder).To(HaveHTTPStatus(http.StatusBadRequest))
+// // 		Expect(response).To(MatchFields(IgnoreMissing, Fields{
+// // 			"MessageID": Equal("E-0007"),
+// // 			//nolint:lll // No need to fix
+// // 			"Message":    Equal("Cannot create the show. Please try again. If the problem continues, kindly reach out to the system administrator for support"),
```

golangci-lint (1.61.0)

the line is 157 characters long, which exceeds the maximum of 120 characters.

(lll)